powertools-metrics/src/main/java/software/amazon/lambda/powertools/metrics/MetricsUtils.java

```diff
@@ -26,6 +29,29 @@ public static MetricsLogger metricsLogger() {
         return metricsLogger;
     }
 
+    /**
+     * Add and immediately flush a single metric. It will use the default namespace
+     * specific either on {@link Metrics} annotation or via POWERTOOLS_METRICS_NAMESPACE env var.
+     *
+     * @param name   the name of the metric
+     * @param value  the value of the metric
+     * @param unit   the unit type of the metric
+     * @param logger the MetricsLogger
+     */
+    public static void withSingleMetricOnDefaultNameSpace(final String name,
```

Comment:
This is great. Really appreciate that.
I'd say that maybe we can expand this to also set the `AwsRequestId` property. Probably we should do the same for `withSingleMetric`.

If we do the `AwsRequestId` part that will ensure that all metrics `metrcisLogger()` and `withSingleMetric` methods from `MetricsUtils` have the request Id and the traceId which ensures good observability and tracing

I know you've taken a very explicit approach in naming the function, but I think the `OnDefaultNameSpace` suffix can be dropped in favour of documentation calling out the usage of the default namespace.

Reply:
I kind of like the idea here. Appreciate the feedback.

Reply:
Suggestion incorporated. Now it will capture request id, trace id whenever possible as property